android/src/main/kotlin/com/gdelataillade/alarm/alarm/BootReceiver.kt

```diff
@@ -28,62 +19,20 @@ class BootReceiver : BroadcastReceiver() {
     private fun rescheduleAlarms(context: Context) {
```

Comment:
I ran a few tests on my Android device and it looks like alarms are not being rescheduled after device reboot.

Reply:
This is the console logs after device reboot on Pixel 8 API 35:

```
2024-11-26 21:55:16.607  3165-3165  BootReceiver            com...ataillade.alarm.alarm_example  I  Device rebooted: action is: android.intent.action.BOOT_COMPLETED
2024-11-26 21:55:16.607  3165-3165  BootReceiver            com...ataillade.alarm.alarm_example  D  Device rebooted, rescheduling alarms
2024-11-26 21:55:16.618  3165-3165  BootReceiver            com...ataillade.alarm.alarm_example  D  Rescheduling 0 alarms
```

As you can see the BroadcastReceiver is running but no alarms are found in storage.

This is likely because the `AlarmStorage.saveAlarm` function is never called. Ever before the migration.

I've added a TODO for you so we don't forget to address this bug.